labs/lab9/9.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Lab 9 - Web Application
+layout: lab
+---
+
+## Overview
+
+We’re going to set up a flask app. And we are going to call it [basicapp]
+
+There is 3 main component to this lab. We will install Nginx, uWSGI, and flask through apt. Then we will git clone an example app and config files from my repository. Next, we will modify the configuration file. If configured correctly, we'll go through the startup procedure. We'll end it with some debugging tips and tricks. 
+
+## Note
+
+* No sudo was included in the commands. Use your best judgment to include sudo or not. 
+* Replace [vars] with your config value 
+
+### The Setup
+
+We need virtualenv and git for staging. Run
+
+`apt install virtualenv git`
+
+Next, we'll create the directory where our project will be located. We'll just call it [basicapp] and we'll place it in /opt/. 
+
+`mkdir /opt/basicapp`
+
+Next, we'll set up our virturalenv. Virtualenv is a python tool that lets you create a custom python execution environment that is specific to your project. This is so dependencies or modified libraries does not pollute your main coding environment. Let's create the virtualenv for this project and activate it. 
+
+`cd /opt/[basicapp]
```

Comment:
This is kind of confusing, I would just change all references of `[basicapp]` to `basicapp` like you stated before in `mkdir /opt/basicapp`

Reply:
Yeah, ok. It was experimental to make instructions more clear. And I wasn't consistent enough. 

Also, note to self, directory instructions is wrong, since it clones from this repository now.

Reply:
And git just don't work that way.